.github/scripts/check-cmake.sh

```diff
@@ -36,7 +36,8 @@ if ! type cmake-format 2> /dev/null ; then
 fi
 
 find . -type d \( \
-    -path ./\*build -o \
+    -path ./\*build\* -o \
+    -path ./release \
```

Comment:
Shouldn't there be another -o after release?

Reply:
Fixed